gadgets/src/integers/uint/tests/uint128.rs

```diff
@@ -394,18 +393,6 @@ fn test_uint128_div() {
         assert!(cs.is_satisfied());
         assert!(r.value == Some(expected));
         check_all_allocated_bits(expected, r);
-
-        // Flip a bit_gadget and see if the division constraint still works
-        if cs
-            .get("division/r_sub_d_result_0/allocated bit_gadget 0/boolean")
-            .is_zero()
-        {
-            cs.set("division/r_sub_d_result_0/allocated bit_gadget 0/boolean", Fr::one());
-        } else {
-            cs.set("division/r_sub_d_result_0/allocated bit_gadget 0/boolean", Fr::zero());
-        }
-
-        assert!(!cs.is_satisfied());
```

Comment:
Perhaps we could comment this out (including the reasoning for doing it) instead of removing it, just in case the `TestConstraintSystem` eventually becomes fast enough to use on `u128`?